Multiple endpoints

degemer opened the pull request.

olivielpeau commented on lib/dogapi/v1/comment.rb:
```diff
@@ -9,36 +9,24 @@ class CommentService < Dogapi::APIService
-        rescue Exception => e
-          suppress_error_if_silent e
+        if options.empty?
+          raise "Must update something."
```
Technically changes the behavior here as this exception won't be rescued anymore

olivielpeau replied:
(we could remove this conditional altogether, it's not really consistent to check this in this method when all the other methods make zero checks)

olivielpeau commented on lib/dogapi/v1/comment.rb:
```diff
@@ -9,51 +9,24 @@ class CommentService < Dogapi::APIService
-        rescue Exception => e
-          suppress_error_if_silent e
+        if options.empty?
+          raise "Must update something."
```
Re-adding my comment here: this exception is not rescued anymore, I think we should remove it altogether for consistency

degemer replied:
Thanks, I'll remove it.

olivielpeau commented:
Looks good! Lots of 💚  for all the nice tests!

Have you looked into the problem that rubocop detects (`unnecessary_double_quotes`)? I think we could ignore it

degemer commented:
Thanks for the review ! I addressed your comment, and added something else: when one endpoint fails, it doesn't affect the other. Could you please have one more look ? 🙇

olivielpeau commented:
I had missed that, looks better now indeed! 🚀 

Feel free to merge! (once the rubocop errors are sorted out ;) )

degemer commented:
`unnecessary_double_quotes` is actually a `tailor` warning. I added `rubocop` for `spec/`, but I'm not going to enable it for `lib/` for now (because there would be too many "errors"). It'll be for later.

I'm going to ignore this `tailor` warning, and merge!
